tests/gumdropTree.ts

```diff
@@ -0,0 +1,77 @@
+import { keccak_256 } from 'js-sha3';
+
+export class MerkleTree {
```

Comment:
I really like this class and how it seems to define how the test is built.  If the test were able to be written without this object, I think the test wouldn't have been as clean.

Reply:
All credit to @lwus